javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutNulls */
+function filterOutNulls(values) {
+  var noNulls = [];
+  for (var i = 0; i < values.length; i++) {
+    if (values[i] != null) {
+      var addValue = values[i];
+      noNulls.push(addValue);
+    }
+  }
+  return noNulls;
+}
```

Comment:
I know this wasn't in the testing suite but let's see if we can fix this issue. This function is also filtering out `undefined`. Since `undefined` is not `null`, let's fix it so that it doesn't filter out `undefined`.

For example:
```
filterOutNulls([null, 1, undefined]); 
```
This should return `[1, undefined]` instead of `[1]`